ci/L0_multi_gpu/multi_lora/multi_lora_test.py

```diff
@@ -119,7 +119,6 @@ def _test_vllm_model(
         self.triton_client.stop_stream()
 
     def test_multi_lora_requests(self):
-        self.triton_client.load_model(self.vllm_model_name)
```

Comment:
Do we know why it doesn't work with explicit mode?

Reply:
Due to the way vllm evaluates free GPU memory. I'm not sure what exactly changed, but in explicit mode `initialize` fails at:
```
 Error in memory profiling. This happens when the GPU memory was not properly cleaned up before initializing the vLLM instance.
 ```
Unfortunately, I don't have the capacity to investigate what exactly changed in the profiling

Reply:
@oandreeva-nv  Have you created ticket for further investigation?

Reply:
DLIS-7012